tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -338,30 +339,85 @@ export interface moveOrCopyResourceArgs {
   resource: string
   newLocation: string
   action: 'copy' | 'move'
+  method: string
+}
+export const pasteResource = async (
+  args: Omit<moveOrCopyResourceArgs, 'action' | 'method'>
+): Promise<void> => {
+  const { page, resource, newLocation } = args
+
+  await page.locator(breadcrumbRoot).click()
+  const newLocationPath = newLocation.split('/')
+
+  for (const path of newLocationPath) {
+    if (path !== 'Personal') {
+      await clickResource({ page, path: path })
+    }
+  }
+
+  await page.locator(clipboardBtns).first().click()
```

Comment:
Are there any cases where there is more than one same element on a page?

Reply:
`ClipboardBtns` element  give us div under which two button bundled
![Screenshot from 2023-01-03 10-18-18](https://user-images.githubusercontent.com/54478846/210300523-8f7bfe4a-f95b-4f07-a00f-444cb3a0fd9b.png)
Thanks, I can didn't noticed that. I will change the selector